test/checks/navigation/region.js

```diff
@@ -74,9 +66,75 @@ describe('region', function () {
 	});
 
 	it('should return false when there is a non-skiplink', function () {
-		fixture.innerHTML = '<div id="target"><a href="something.html#mainheader">Click Here</a><div role="main"><h1 id="mainheader">Introduction</h1></div></div>';
-		var node = fixture.querySelector('#target');
-		assert.isFalse(checks.region.evaluate.call(checkContext, node));
+		var checkArgs = checkSetup('<div id="target"><a href="something.html#mainheader">Click Here</a><div role="main"><h1 id="mainheader">Introduction</h1></div></div>');
+
+		assert.isFalse(checks.region.evaluate.apply(checkContext, checkArgs));
 		assert.equal(checkContext._relatedNodes.length, 1);
 	});
+
+	it('should return true if the non-region element is a script', function () {
+		var checkArgs = checkSetup('<div id="target"><script>axe.run()</script><div role="main">Content</div></div>');
+
+		assert.isTrue(checks.region.evaluate.apply(checkContext, checkArgs));
+	});
+
+	it('should considered aria labelled elements as content', function () {
+		var checkArgs = checkSetup('<div id="target"><div aria-label="axe-core logo"></div><div role="main">Content</div></div>');
```

Comment:
Shouldn't this DIV have a role on it for `aria-label` to be considered content? It isn't exposed consistently for non-interactive elements or landmarks in my experience